### PR TITLE
[GNA] added split_to_concat_multi_inputs test

### PR DESCRIPTION
**Pull request by a-noskov**
### Details:
 - added split_to_concat_multi_inputs tests with GNA_SW_FP32 from tests_deprecated/fp32_non_quantized_tests

### Tickets:
 - 62542

### CODE REVIEW

File: inference-engine/tests/functional/shared_test_classes/src/subgraph/split_concat_multi_inputs.cpp [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    }
+    return result.str();
+}
+
+void SplitConcatMultiInputsTest::SetUp() {
+    std::vector<size_t> inputShape;
+    size_t splitsNum;
+    std::map<std::string, std::string> tempConfig;
+    InferenceEngine::Precision netPrecision;
+    //std::string targetDevice;
```
Comment (nkogteva) [inline inference-engine/tests/functional/shared_test_classes/src/subgraph/split_concat_multi_inputs.cpp:35]: Uncomment this. I'm wondering how it is working? It seems CI doesn't run GNA)

File: inference-engine/tests/functional/shared_test_classes/src/subgraph/split_concat_multi_inputs.cpp [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    configuration.insert(tempConfig.begin(), tempConfig.end());
+
+    inputShape[1] *= splitsNum;
+    auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
+    auto params = ngraph::builder::makeParams(ngPrc, {inputShape});
+
+    auto split = ngraph::builder::makeSplit(params[0], ngPrc, splitsNum, 1);
+    ngraph::OutputVector concatInputs = split->outputs();
+
+    auto concat = std::make_shared<ngraph::opset1::Concat>(concatInputs, 1);
```
Comment (nkogteva) [inline inference-engine/tests/functional/shared_test_classes/src/subgraph/split_concat_multi_inputs.cpp:47]: Use latest opset

File: inference-engine/tests/functional/shared_test_classes/src/subgraph/split_concat_multi_inputs.cpp (hunk 1 above)
Comment (a-noskov) [reply inference-engine/tests/functional/shared_test_classes/src/subgraph/split_concat_multi_inputs.cpp:35]: targetDevice is initialized in base class LayerTestsCommon, so I will delete this

File: inference-engine/tests/functional/plugin/gna/shared_tests_instances/subgraph_tests/split_concat_multi_inputs.cpp [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+using namespace SubgraphTestsDefinitions;
+
+namespace {
+std::vector<InferenceEngine::Precision> precisions = {InferenceEngine::Precision::FP32,
+                                                         InferenceEngine::Precision::FP16};
+
+std::vector<std::map<std::string, std::string>> additionalConfig = {
+    {
+        {"GNA_DEVICE_MODE", "GNA_SW_FP32"}
```
[the remark this answers is not in the document]
Comment (a-noskov) [reply inference-engine/tests/functional/plugin/gna/shared_tests_instances/subgraph_tests/split_concat_multi_inputs.cpp:20]: Added GNA_SW_EXACT mode

**Review by nkogteva — APPROVED** [review]

**Review by mryzhov — APPROVED** [review]

**Pull request merged**